packages/mui-material/src/Tabs/TabsContext.ts

```diff
@@ -0,0 +1,36 @@
+'use client';
+import * as React from 'react';
+import type { TabProps } from '@mui/material/Tab';
+import type { TabsProps } from './Tabs';
+
+interface TabsContextType {
+  fullWidth?: boolean;
+  indicator?: React.JSX.Element;
+  mounted?: boolean;
+  selectionFollowsFocus?: TabsProps['selectionFollowsFocus'];
+  onChange?: TabsProps['onChange'];
+  textColor?: TabsProps['textColor'];
+  tabsValue?: TabsProps['value'];
```

Comment:
```suggestion
  value?: TabsProps['value'];
```
I think it's better to use `value` here, it's a standard name with `onChange`. The `TabsContext` will need to be exposed to public.

Reply:
Done.